go.mod

```diff
@@ -80,7 +80,7 @@ require (
 	github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d // indirect
 	go.opencensus.io v0.24.0 // indirect
 	golang.org/x/arch v0.7.0 // indirect
-	golang.org/x/net v0.23.0 // indirect
+	golang.org/x/net v0.24.0 // indirect
```

Comment:
Hi @Alonza0314 ,

Please update net module to version `v0.33.0`.
Thanks!

Reply:
OK! I will update it in an another new PR.